### PR TITLE
feat: Landing page

### PR DESCRIPTION
**Pull request by Pedrobourguignon**
- [x] Create Cali Landing Page

### CODE REVIEW

File: components/Button/PrimaryButton.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { Button, Flex } from '@chakra-ui/react'
+import { usePicasso } from 'hooks/usePicasso'
+
+export const PrimaryButton = () => {
+	const theme = usePicasso()
+	return (
+		<Flex boxShadow="0px 0px 10px rgba(232, 49, 81, 0.5)" mt="16">
```
Comment (ThalesBMC) [inline components/Button/PrimaryButton.tsx:7]: Please put the boxShadow in chakra theme

File: components/Header/Header.tsx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+								<MenuItem>
+									<NextLink href="/">
+										<Link>Docs</Link>
+									</NextLink>
+								</MenuItem>
+								<MenuItem>
+									<NextLink href="/">
+										<Link>Contact</Link>
+									</NextLink>
+								</MenuItem>
```
Comment (ThalesBMC) [inline components/Header/Header.tsx:102]: Transform  this in a component but put inside this file and do not create other file for it.
So you can call it with  arrayOfNames.map 
```
<MenuItem>
							<NextLink href="/">
								<Link>Docs</Link>
				</NextLink>
</MenuItem>
```

File: components/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+				</NextLink>
+				<NextLink href="/">
+					<Link>Roadmap</Link>
+				</NextLink>
+				<NextLink href="/">
+					<Link>Docs</Link>
+				</NextLink>
+				<NextLink href="/">
+					<Link>Contact</Link>
+				</NextLink>
```
Comment (ThalesBMC) [inline components/Header/Header.tsx:46]: Transform this in a component but put inside this file and do not create other file for it.
So you can call it with arrayOfNames.map

File: layouts/Background/Background.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+interface IBackground {
+	children: React.ReactNode
+}
+
+export const Background: React.FC<IBackground> = ({ children }) => {
+	return (
+		<Flex
+			direction="column"
+			w="full"
+			background="linear-gradient(112.87deg, rgba(26, 29, 225, 0.1) 0%, rgba(16, 15, 18, 0.1) 32.29%, rgba(16, 15, 18, 0.1) 66.15%, rgba(2, 228, 209, 0.1) 100%);"
```
Comment (ThalesBMC) [inline layouts/Background/Background.tsx:13]: put this on a chakra variable

File: pages/index.tsx [hunk 5]
```diff
@@ -1,8 +1,60 @@
...
+					<Flex ml="4">
+						“
+						<Flex
+							borderBottomWidth="0.25rem"
+							borderBottomStyle="solid"
+							borderColor={theme.branding.blue}
+						>
+							time is money
+						</Flex>
+						.”
```
Comment (ThalesBMC) [inline pages/index.tsx:31]: If you are using this “ “ please put inside a text

**Review by ThalesBMC — CHANGES_REQUESTED** [review]

**Review by ThalesBMC — APPROVED** [review]

File: pages/index.tsx [hunk 6]
```diff
@@ -1,6 +1,60 @@
-import { Flex } from '@chakra-ui/react'
+import { Flex, Icon, Text } from '@chakra-ui/react'
+import { PrimaryButton } from 'components/Button/PrimaryButton'
+import { Background } from 'layouts/Background/Background'
```
Comment (amaurymagalhaes) [inline pages/index.tsx:3]: This should be imported from a barrel like  
```ts
import { Background } from 'layouts'
```

**Review by amaurymagalhaes — CHANGES_REQUESTED** [review]

**Pull request merged**